src/index.html

```diff
@@ -1,14 +1,160 @@
+  resort, to rebellion against tyranny and oppression, that human rights should be
+  protected by the rule of law,
+  Whereas it is essential to promote the development of friendly relations between
+  nations,
+      <br>
+  Whereas the peoples of the United Nations have in the Charter reaffirmed their
+  faith in fundamental human rights, in the dignity and worth of the human person
+  and in the equal rights of men and women and have determined to promote
+  social progress and better standards of life in larger freedom,
+  Whereas Member States have pledged themselves to achieve, in cooperation
+  with the United Nations, the promotion of universal respect for and observance of
+  human rights and fundamental freedoms,
+      <br>
+  Whereas a common understanding of these rights and freedoms is of the
+  greatest importance for the full realization of this pledge,
+  Now, therefore,
+      <br>
+  The General Assembly,
+      <br>
+  Proclaims this Universal Declaration of Human Rights as a common standard of
+  achievement for all peoples and all nations, to the end that every individual and
+  every organ of society, keeping this Declaration constantly in mind, shall strive by
+  teaching and education to promote respect for these rights and freedoms and by
+  progressive measures, national and international, to secure their universal and
+  effective recognition and observance, both among the peoples of Member States
+  themselves and among the peoples of territories under their jurisdiction.
+    </p>
+    <h2>Article 2</h2>
+    <p>Everyone is entitled to all the rights and freedoms set forth in this Declaration,
+      without distinction of any kind, such as race, colour, sex, language, religion,
+      political or other opinion, national or social origin, property, birth or other status.
+      Furthermore, no distinction shall be made on the basis of the political,
+      jurisdictional or international status of the country or territory to which a person
+      belongs, whether it be independent, trust, non-self-governing or under any other
+      limitation of sovereignty.
+    </p>
+    <h2>Article 3</h2>
+    <p>Everyone has the right to life, liberty and the security of person.
+
+    </p>
+    <h2>Article 4</h2>
+    <p>No one shall be held in slavery or servitude; slavery and the slave trade shall be
+      prohibited in all their forms.
+    </p>
+    <h2>Article 5 </h2>
+    <p>No one shall be subjected to torture or to cruel, inhuman or degrading treatment
+      or punishment.
+    </p>
+    <h2>Article 6 </h2>
+    <p>Everyone has the right to recognition everywhere as a person before the law.
+
+    </p>
+    <h2>Article 7 </h2>
+    <p>All are equal before the law and are entitled without any discrimination to equal
+      protection of the law. All are entitled to equal protection against any
+      discrimination in violation of this Declaration and against any incitement to such
+      discrimination.
+    </p>
```

Comment:
```suggestion
    <p>
      All are equal before the law and are entitled without any discrimination to equal
      protection of the law. All are entitled to equal protection against any
      discrimination in violation of this Declaration and against any incitement to such
      discrimination.
    </p>
```
отредактируй во всех местах похожую ошибку